msteams-platform/toolkit/add-capability.md

```diff
@@ -96,16 +97,15 @@ After adding tab, the changes in your project are as follows:
 - The files under `templates/azure/teamsfx` will be updated, and `templates/azure/provision/xxx`.bicep file will be regenerated.
 - The file under `.fx/config` are regenerated, which ensures your project is set with right configurations for newly added capability.
 
-## Limitations
-
-The limitations with TeamsFx while adding more capabilities are as follows:
+## Limitation
 
+The limitation to TeamsFx while adding more capabilities is as follows:
 - You can't add project capability for more than one instance.
```

Comment:
This isn't entirely accurate; tabs are allowed up to 16 instances. Bot / Messaging Extension are allowed one instance each.